pages/linux/betterlockscreen.md

```diff
@@ -0,0 +1,19 @@
+#betterlockscreen
+
+>simple, minimal lock screen
+
+- Lock the screen:
+
+`betterlockscreen --lock`
+
+-Change lock screen background:
+
+`betterlockscreen -u path/to/the/image.png`
+
+-Lock the screen with custom text:
+
+`betterlockscreen -l pixel -t "custom lock screen text"`
```

Comment:
Perhaps we could use the `{{token_syntax}}` here?
```suggestion
`betterlockscreen -l pixel -t "{{custom lock screen text}}"`
```

Also, are there any long forms of these arguments? In tldr pages, we prefer the longer form options as it aids clarity.